assets/js/blocks/product-gallery/edit.tsx

```diff
@@ -71,6 +72,7 @@ const TEMPLATE: InnerBlockTemplate[] = [
 										type: 'flex',
 										verticalAlignment: 'bottom',
 									},
+									lock: { move: true, remove: true },
```

Comment:
Is it expected that the Next/Prev buttons have a removal lock as well? I think it makes sense, but still making sure it's added on purpose.

Reply:
Yes, this is on purpose.

I have tried to explain it in the PR description :)
>- The lock on the Sale Badge still allows the user to delete the block.
>- For the Next/Prev Buttons such thing is not necessary as the user can hide the block with block settings.